README.md

```diff
@@ -497,6 +497,19 @@ Changing the theme.
 negotiator.switchTheme("dark");
 ```
 
+### Tokens Updated Hook
+
+Detect when new tokens have been added to the Token Negotiator during an applications lifecycle (for off chain tokens at this time).
+
+```javascript
+
+// temporary solution likely to change in the next major release version.
+document.body.addEventListener("tokensupdated", () => {
+	console.log("Tokens updated event fired!!");
```

Comment:
we have to add new negotiate(), because tokens list changed after that action

Reply:
Thanks @oleggrib, I think this should be ok for the README.

Reply:
Where the end developer may wish to use this event for another task. E.g. Read the new list of tokens and navigate them to a gated page location.